build.ps1

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env pwsh
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+param(
+    [Parameter()]
+    [switch]
+    $Bootstrap,
```

Comment:
Looking at your logic below on lines 101-124, it seems that `-Bootstrap` should be it's own ParameterSet as it's mutually exclusive to `-Clean` and `-Test`?

Reply:
got it